projects/commudle-admin/src/app/feature-modules/user-follow/user-follow.component.html

```diff
@@ -0,0 +1,25 @@
+<div *ngIf="user.is_expert" class="user-follow-container">
```

Comment:
let's name this class same as the component name `user-follow`

Reply:
Done!